src/main/java/cardibuddy/ui/MainWindow.java

```diff
@@ -106,13 +106,13 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
-        personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+        personListPanel = new PersonListPanel(logic.getFilteredFlashcardList());
         personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
```

Comment:
Not sure if you want to change this since Sigmund is editing MainWindow also but I think the code should be 'getFilteredDeckList()'. For flashcards, I think it would be under the CommandResult section that would call the open method to show all the flashcards?

Reply:
ps. I'm not changing this part so go ahead and make any changes!

Reply:
I haven't really changed the UI part cause there are other dependencies (PersonListPanel should be changed to DeckListPanel and it'll take a while!) I'll do it on the next iteration